ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/AbstractJsonRpcHttpBySpecTest.java

```diff
@@ -50,6 +52,13 @@
 public abstract class AbstractJsonRpcHttpBySpecTest extends AbstractJsonRpcHttpServiceTest {
 
   private static final ObjectMapper objectMapper = new ObjectMapper();
+  private static final Pattern GAS_MATCH_FOR_STATE_DIFF =
```

Comment:
@pinges  and cc @macfarla 
I personally don't like this approach because:
1. Using regex we're removing a lot of additional gas related fields that are not causing discrepancies, this opens doors for future regression
2. I think fixing the json responses generated by OE and documenting how we've done it is a more precise and lower touch approach, I just need to get this to build locally https://github.com/frankisawesome/openethereum/pull/1/files

Reply:
Maybe I did something wrong, using my OE fork with the changes I'm still getting the same JSONs from the conformance test repo :(

Reply:
agree I would go with fixing up the json responses that we get from OE

Reply:
https://github.com/ConsenSys/json-rpc-conformance-test-utils/issues/20

Reply:
Do I understand this correctly that for now, we have to use regex because of the differences with OE, but once the [linked issue](https://github.com/ConsenSys/json-rpc-conformance-test-utils/issues/20) is resolved we could get rid of it?